### ISSUE TITLE
Python 2.7 doctest failures on 64-bit Windows

### ISSUE THREAD

**Issue by matthew-brett** [nipy/dipy#812]
Around 7 failures of form:

```
Failed example:
    data.shape
Expected:
    (58, 58, 24)
Got:
    (58L, 58L, 24L)
```

See: http://nipy.bic.berkeley.edu/builders/dipy-bdist64-27/builds/9/steps/shell_11/logs/stdio

These are obviously caused by the fact that the values are `npy_intp` type, a 64-bit integer, which is larger than the standard 32-bit integer on Windows, and therefore gets returned as a Python long integer.

The most obvious fix is to replace stuff like:

```
>>> data.shape
(58, 58, 24)
```

With:

```
>>> data.shape == (58, 58, 24)
True
```

**Comment by omarocegueda** [comment]
+1, I can do that if you haven't started yet

**Comment by matthew-brett** [comment]
Thank you - no I haven't started.

**Comment by arokem** [comment]
closed through #814
